server/package.json

```diff
@@ -17,6 +17,7 @@
     "seed:superuser": "env $(cat .env) ts-node scripts/superuser.ts",
     "seed:mapping": "env $(cat .env) ts-node scripts/importMapping.ts",
     "seed:definitions": "env $(cat .env) ts-node scripts/fhirDefinitions.ts $(ls fhir/*.json)",
+    "export:mapping": "env $(cat .env) ts-node scripts/exportMapping.ts",
```

Comment:
I would call it "export:mappings" (plural) since we consider that each source has its own mapping.
Also, I would add this command before the "yarn migrate"

Reply:
yes